phpseclib/Crypt/TripleDES.php

```diff
@@ -267,6 +265,32 @@ function setIV($iv)
     }
 
     /**
+     * Sets the key length.
+     *
+     * Valid key lengths are 64, 128 and 192
+     *
+     * @see Crypt_Rijndael:setKeyLength()
```

Comment:
I think this should be `@see Crypt_Base:setKeyLength()` here and `@see \phpseclib\Crypt\Base::setKeyLength()` in 2.0.